packages/pages/src/components/analytics/provider.tsx

```diff
@@ -22,12 +22,17 @@ export function AnalyticsProvider(
     enableTrackingCookie,
     enableDebugging,
     templateData,
+    pageDomain,
   } = props;
 
   const analyticsRef = useRef<AnalyticsMethods | null>(null);
 
   if (analyticsRef.current === null) {
-    analyticsRef.current = new Analytics(templateData, requireOptIn);
+    analyticsRef.current = new Analytics(
+      templateData,
+      requireOptIn,
+      pageDomain
```

Comment:
Per your comment
> If none is specified, the hostname for the site ID is used

does that mean the analytics library itself handles the logic and we're just passing this prop through to it? If so, do you know if it handles empty string in a particular way? I could see that leading to something unexpected.

Reply:
This is actually handled by the backend (Fusion). The request is still valid regardless of the string that's passed in (even invalid URL or empty string). But, it seems like the event would be logged as invalid if the `pageDomain` doesn't have a scheme (for `product=sites` only) and there is no hostname for the site ID.

Here we're passing straight to the analytics library and the analytics library checks that the string has a scheme before it sends it as a query param